pkg/securitypolicy/securitypolicyenforcer.go

```diff
@@ -451,8 +447,8 @@ func (pe *StandardSecurityPolicyEnforcer) EnforceOverlayMountPolicy(containerID
 		return errors.New(errmsg)
 	}
 
-	for i, deviceList := range pe.Devices {
-		if equalForOverlay(layerPaths, deviceList) {
+	for i, container := range pe.Containers {
+		if equalForOverlay(incomingOverlay, container.Layers) {
```

Comment:
nit: Could try and find a way to just make this just one loop over seeming as how we're doing the same `equalForOverlay` check again after we should already know the answer.

Reply:
Apparently github didn't feel like showing the right line range, 439 to 451 is what I'm referring to.

Reply:
I ended up "caching" the matching policy containers, but unfortunately can't really get rid of the second loop, since we still need to check that the overlay can be used. We've actually decided to move away from enforcing "exactly one" overlay per container entry, but I think it makes sense to introduce that change in a separate PR.